### ISSUE TITLE
Package Manager Question

### ISSUE THREAD

**Issue by rnc** [release-engineering/cachito#353]
https://github.com/release-engineering/cachito#feature-support lists package managers of `gomod | npm | pip` but https://github.com/release-engineering/cachito#git-submodule implies git-submodule is also a package manager. Is the table correct?

**Comment by rnc** [comment]
@lcarva  Could you take a look at this please?

**Comment by lcarva** [comment]
Filed the internal tracker as CLOUDBLD-3792

**Comment by lcarva** [comment]
`git-submodule` is a package manager from Cachito's perspective.

The way to use it is by listing `git-submodule` in the `pkg_managers` list of a Cachito request. You may use `git-submodule` with other package managers. For now, the only restriction is using `git-submodule` together with `gomod`. 

When the `git-submodule` package manager is used, Cachito will also fetch the contents of the git repos referenced as submodules. However, if a submodule in turn has submodules, those will not be fetched.

Does that answer your immediate questions until we can get this information incorporated into the docs?

**Comment by rnc** [comment]
@lcarva Yes thanks, it was originally for the work on https://github.com/cekit/cekit/issues/663
